packages/round-manager/src/features/round/CreateRoundPage.tsx

```diff
@@ -50,8 +48,8 @@ export default function CreateRound() {
           <main>
             <FormWizard
               steps={[
-                RoundDetailForm,
-                ApplicationEligibilityForm,
+                // RoundDetailForm,
```

Comment:
```suggestion
                RoundDetailForm,
```

Reply:
I comment them out to test, was going to put back after.